ocs_ci/utility/templating.py

```diff
@@ -134,9 +134,48 @@ def dump_to_temp_yaml(src_file, dst_file, **kwargs):
         yaml.dump(data, yaml_file)
 
 
-def load_yaml_to_dict(file):
+def load_yaml_to_dict(file, multi_document=False):
+    """
+    Load yaml file to the dictionary
+
+    Args:
+        file (str): Path to yaml file to load
+        multi_document (bool): True if yaml contains more documents
```

Comment:
do you mean here that a single yaml contain few CRs inside it?
if yes, please add this to the description

Reply:
The description follows yaml conventions:
https://yaml.org/spec/1.2/spec.html

YAML uses three dashes (“---”) to separate directives from document content.

CR is openshift specific but here we are working with pure yaml.